File: atcoder/abc142C_go_to_school.py
# Vicfred
# https://atcoder.jp/contests/abc142/tasks/abc142_c
# implementation
n = int(input())
a = list(map(int,input().split()))

students = list()
for i in range(1,n+1):
    students.append((i,a[i-1]))

#print(sorted(students, key=lambda x: x[1]))
students.sort(key=lambda x: x[1])

for student in students:
    print(student[0], end=' ')
print()

File: atcoder/abc142C_go_to_school.ml
(* vicfred
 * https://atcoder.jp/contests/abc142/tasks/abc142_c
 * implementation, simulation
 * *)
Scanf.scanf "%d\n" @@ fun n ->
    let a = Array.init n (fun i -> Scanf.scanf "%d " (fun a -> i+1,a)) in
    Array.sort (fun (a,b) (x,y) -> compare b y) a;
    Array.iter (fun (x, _) -> Printf.printf "%d " x) a;
    Printf.printf "\n"

